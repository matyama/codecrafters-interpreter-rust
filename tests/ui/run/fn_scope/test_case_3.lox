var counter = 99;
{
  counter = counter + 5;
  {
    counter = counter + 5;
    print counter;
  }
}
print counter;
{
  var counter = 99;
  print counter;
}
print counter;
