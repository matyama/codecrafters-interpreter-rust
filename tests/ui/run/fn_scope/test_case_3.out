109
109
99
109
